plus/async/sleep.js

```diff
@@ -2,6 +2,22 @@
  * It wait for the given seconds, but at most 10 seconds.
  *
  * @param {number} sec the seconds to wait
- * @returns {Promise} the promise which resolves when wait if over
+ * @returns {Promise} the promise which resolves when wait is over
  *                    or rejects if parameter is not correct
- */
\ No newline at end of file
+ */
+module.exports = function wait(sec) {
+    return new Promise((resolve, reject) => {
+        if (typeof sec !== "number") {
+            reject(Error("sec has to be a number"));
+        }
+        if (sec < 1) {
+            reject(Error("sec can not be less than 1"));
+        }
+        if (sec > 10) {
+            sec = 10;
```

Comment:
Please reject the function with the proper error message if the sec is higher than 10.

Reply:
If I reject it with an error, the second test case for the function will fail.